src/vs/workbench/contrib/search/browser/searchView.ts

```diff
@@ -1538,6 +1540,30 @@ export class SearchView extends ViewletPanel {
 		}, errors.onUnexpectedError);
 	}
 
+	openEditorWithMultiCursor(element: FileMatchOrMatch): Promise<void> {
+		const selection = this.getSelectionFrom(element);
+		const resource = element instanceof Match ? element.parent().resource : (<FileMatch>element).resource;
+		return this.editorService.openEditor({
+			resource: resource,
+			options: {
+				preserveFocus: false,
+				pinned: true,
+				selection,
```

Comment:
Is this selection necessary since you select everything in the next block?

Reply:
I see, I was misunderstanding. I think this isn't exactly what we want then because it should select all of the matches from the search view, not "all occurrences of what is selected right now".

For example, all the matches could be different if you are matching a regex, and some identical text in a different place in the document may not actually match. Does this work without the second block at all?

Reply:
This does not work for regex. I'll fix it with a commit in a few days :)